### PR TITLE
Add RPC methods

### PR DESCRIPTION
**Pull request by akegaviar**
In lieu of [PR#456](https://github.com/chainstack/docs/pull/456)

### CODE REVIEW

**Comment by akegaviar** [comment]
@py-goh done

File: docs/api/node-json-rpc-api.md [hunk 1]
```diff
@@ -0,0 +1,1824 @@
...
+    content: Collection of JSON-RPC methods available with examples in cURL, web3.py, web3.js.
+  - name: keywords
+    content: json rpc methods curl api web3.py web3.js
+---
+
+# JSON-RPC API reference
+
+This page is a collection of JSON-RPC API call examples for Ethereum-based nodes with [web3.js](https://web3js.readthedocs.io/), [web3.py](https://web3py.readthedocs.io/), and [cURL](https://curl.se/).
+
+For a detailed description of all the available JSON-RPC methods, see [Ethereum Wiki](https://eth.wiki/json-rpc/API)
```
Comment (easeev) [inline docs/api/node-json-rpc-api.md:13]: Full stop missing?

File: docs/api/node-json-rpc-api.md [hunk 2]
```diff
@@ -0,0 +1,1824 @@
...
+meta:
+  - name: description
+    content: Collection of JSON-RPC methods available with examples in cURL, web3.py, web3.js.
+  - name: keywords
+    content: json rpc methods curl api web3.py web3.js
+---
+
+# JSON-RPC API reference
+
+This page is a collection of JSON-RPC API call examples for Ethereum-based nodes with [web3.js](https://web3js.readthedocs.io/), [web3.py](https://web3py.readthedocs.io/), and [cURL](https://curl.se/).
```
Comment (easeev) [inline docs/api/node-json-rpc-api.md:11]: >Ethereum-based

Is it consistent with other docs? I.e. Ethereum-based or EVM-based?

**Review by easeev — CHANGES_REQUESTED** [review]
See comments

File: docs/.vuepress/config.js [hunk 3]
```diff
@@ -353,7 +353,8 @@ module.exports = {
                 children: [
                     '/api/',
                     '/api/quick-tutorial',
-                    [`https://${process.env.DOCS_HOSTNAME}/api/reference/`, 'API reference'],
+                    [`https://${process.env.DOCS_HOSTNAME}/api/reference/`, 'Platform API reference'],
+                    '/api/node-json-rpc-api',
```
[the remark this answers is not in the document]
Comment (akegaviar) [reply docs/.vuepress/config.js:357]: > in the menu doesn't look good. Platform API could be technically also JSON-RPC API so it's confusing

Changed.

> Solana also has JSON-RPC API but we don't have it here. Will we have it BTW.

Yes, it's planned for the next iteration.

File: docs/api/node-json-rpc-api.md (hunk 2 above)
Comment (akegaviar) [reply docs/api/node-json-rpc-api.md:11]: Changed.

File: docs/api/node-json-rpc-api.md (hunk 1 above)
Comment (akegaviar) [reply docs/api/node-json-rpc-api.md:13]: Fixed.

File: docs/api/node-api-reference.md [hunk 4]
```diff
@@ -0,0 +1,2126 @@
+---
+meta:
+  - name: description
+    content: Collection of JSON-RPC methods available with examples in web3.js, web3.py, eth.rb, and cURL.
```
Comment (easeev) [inline docs/api/node-api-reference.md:4]:
```suggestion
    content: Collection of JSON-RPC methods for EVM nodes available with examples in web3.js, web3.py, eth.rb, and cURL.
```

or something similar?

**Pull request merged**